src/TGUI/String.cpp

```diff
@@ -46,7 +46,7 @@ namespace tgui
     /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 #ifdef TGUI_NEXT
-    bool isWhitespace(char32_t character)
+    bool isWhitespace(uint32_t character)
```

Comment:
Shouldn't be changed

Reply:
Error as described before. On compile time.